backend/src/db/seed.py

```diff
@@ -23,6 +23,18 @@ async def main() -> None:
         password=get_password_hash("Test1234!"),
         is_active=True,
         created_at=now,
+        is_admin=True,
+        last_login=datetime.now(),
+    )
+    session.add(user)
```

Comment:
instead of adding the user and admin separately to the DB can you put both user objects in a list and use session.add_all(). Also, make sure that all the other data that gets seeded is linked to that second user and not the admin.